README.md

```diff
@@ -70,6 +82,7 @@ The `code` helper supports [Rouge](https://github.com/jayferd/rouge) instance fo
 * `start_line`
 * `css_class`
 * `wrap`
+* `classes`
```

Comment:
Maybe we can rename this to `extra_css_classes`? 🤔 It seems more consistent with the current `css_class` option.

Reply:
I can do that, no problem. I'll work on the update.